numpy/random/_examples/cffi/parse.py

```diff
@@ -17,11 +17,20 @@ def parse_distributions_h(ffi, inc_dir):
                 continue
             s.append(line)
         ffi.cdef('\n'.join(s))
-            
+
     with open(os.path.join(inc_dir, 'random', 'distributions.h')) as fid:
         s = []
         in_skip = 0
+        ignoring = False
```

Comment:
Why do the guards need to be stripped?

Reply:
I can't say (not familiar with cffi), but the tests fail when they're there and pass when they're not.  I assume it's for the same reason that include statements (and friends) are stripped